app/styles/headerBar.css

```diff
@@ -28,6 +59,9 @@
   .socialIcons {
     height: 78px;
   }
+  .nav-link svg {
+    margin-left: 2px; /* Encore plus proche pour les petits écrans */
```

Comment:
Remove unnecessary comments (especially if they are not in English). Here, I believe the css is self-explanatory.